CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
 * [artifact-manager] IAC-796 / Upgrade VCD archetype to support Angular 15
 * [vRA-NG] IAC-782 / vra-ng:pull : Added support for export of policies of the following types: Approval Policy, Day 2 Actions Policy, Deployment Limit Policy, Lease Policy, Resource Quota policy.
 * [artifact-manager] IAC-782 / vrealize:push : Added support for import of policies of the following types: Approval Policy, Day 2 Actions Policy, Deployment Limit Policy, Lease Policy, Resource Quota policy.
-
```

Comment:
Hi @unbreakabl3 , thanks for contributing again! We will check the change next week so we can merge this. 
Can you just move the CHANGELOG.md entry to the top of the file, above the latest release - I think the release commit might have messed with your branch and put this as part of the 2.38.0 changes.

Just move it to the top with a new section (like Enhancements). Here is a sample of a first entry after a release:
https://github.com/vmware/build-tools-for-vmware-aria/pull/233/files